test/integration/viewer_spec.mjs

```diff
@@ -178,15 +178,21 @@ describe("PDF viewer", () => {
     describe("triggers when going bigger than maxCanvasPixels", () => {
       let pages;
 
-      const MAX_CANVAS_PIXELS = 4_000_000;
+      const MAX_CANVAS_PIXELS = new Map();
```

Comment:
Shouldn't all test changes, except for the uncommenting of the previously failing test below, be moved into the first commit instead? That way we have a clear separation where one commit only includes the test, which the failing bit commented out, and one commit that only fixes the found issue and uncomments the failing bit.

Reply:
Oh yes, I forgot I had two commits when changing these files.